### ISSUE TITLE
Animated GIF bug

### ISSUE THREAD

**Issue by markpud** [marijnkampf/Random-Photo-Screensaver#71]
Hi Marijn,

Have found a bug with GIF animation with this option selected:

- Slide images across each monitor in sequence

The GIF animates correctly on monitor 1, but when it moves to monitor 2, it doesn't animate.

Thanks, Mark

**Comment by marijnkampf** [comment]
If there are two different GIF animations on monitor 1 and 2, do they animate then?

**Comment by markpud** [comment]
Yes it seems to work with gif animations on both screens, but when a static image is on monitor 1 then the gif on monitor 2 doesn't animate.
